ui/src/views/Tasks/NoTask/index.jsx

```diff
@@ -48,7 +48,12 @@ export default class NoTask extends Component {
       <Dashboard
         title="View Tasks"
         helpView={<HelpView description={description} />}
-        search={<Search onSubmit={this.handleTaskSearchSubmit} />}>
+        search={
+          <Search
+            placeholder="Search Task ID"
+            onSubmit={this.handleTaskSearchSubmit}
+          />
+        }>
```

Comment:
If you rebase your branch on the current master then this change and the one in NoTaskGroup won't be included in this PR.

Reply:
Rebasing has been done @helfi92.